profiles/default.profile

```diff
@@ -1,2 +1,6 @@
 ENABLE_MODULE_IOTJS_BASIC_MODULES
 ENABLE_MODULE_IOTJS_CORE_MODULES
+ENABLE_MODULE_SAX
+ENABLE_MODULE_DBUS
+ENABLE_MODULE_ZLIB
```

Comment:
We could remove zlib for linux.

Reply:
should we rename the profile to linux.profile or other name? It is difficult to distinguish the meaning of basic.profile and default.profile

Reply:
IMO `default.profile` is a feature rich profile, which would enable most of features that ShadowNode supports like `websocket` and `dbus`. Rather `basic.profile` enables features that are API stable in Node.js.

Reply:
`default.profile` LGTM because the `default.profile` in ShadowNode means ShadowNode itself, but that's better if we change the `basic.profile` to `nodejs.profile` :)

Reply:
And zlib should move to `nodejs.profile` or `basic.profile`, but I dunno where to disable zlib, just resolving because the zlib linking is still required though we disable the zlib module.

Reply:
AFAIK libz doesn't been linked on linux-android.

https://github.com/yodaos-project/ShadowNode/pull/493/files#diff-af3b638bc2a3e6c650974192a53c7291R119